core/src/main/java/com/linecorp/armeria/common/HttpMessageAggregator.java

```diff
@@ -62,6 +62,10 @@ public final void onError(Throwable t) {
 
     @Override
     public final void onComplete() {
+        if (future.isDone()) {
```

Comment:
If `onComplete()` called multiple times, this method throws null pointer exception at 92 line.
contentList is 
```java
// All items in contentList was set to null in the previous `onComplete()` call
final HttpData data = contentList.set(i, null); 
// data is null
final int dataLength = data.length();
```